src/app/modules/worksite/components/CourthouseSelector.tsx

```diff
@@ -15,17 +15,20 @@ export default class CourthouseSelector extends React.PureComponent<FormFieldWra
                     <option value="Courtenay">Courtenay</option>
                     <option value="Duncan">Duncan</option>
                     <option value="Gold River">Gold River</option>
+                    <option value="Kamloops">Kamloops</option>
```

Comment:
Could have these as a constant within the API, at some point they will come from the API.  But again, just a note.  Not needed for the MVP.

Reply:
Good idea. I have included the list of courthouses and regions in the API and updated the selectors to use the new objects